runtime/manual/tools/coverage.md

```diff
@@ -0,0 +1,143 @@
+# deno coverage
+
+Print coverage reports from coverage profiles.
+
+## Command
+
+`deno coverage [OPTIONS] <COVERAGE>`
+
+## Synopsis
+
+```bash
+deno coverage [--ignore=<ignore>] [--include=<regex>] [-q|--quiet] [--exclude=<regex>] [--lcov] [--output=<output>] [--html] [--detailed] [-h|--help] <COVERAGE>
+
+deno coverage -h|--help
+```
+
+## Description
+
+Print coverage reports from coverage profiles.
+
+By default, when you run `deno test --coverage` a coverage profile will be generated in the `/coverage` directory of your workspace.
+Subsequently you can run `deno coverage` to print a coverage report to stdout.
+
+```bash
+deno test --coverage
+deno coverage
+```
+
+## Inclusions and Exclusions
+
+By default coverage includes any of your code that exists on the local file system, and it's imports.
+
+You can customize the inclusions and exclusions by using the `--include` and `--exclude` options.
+
+You can expand the coverage to include files that are not on the local file system by using the `--include` option and customizing the regex pattern.
+
+```bash
+deno coverage --include="^file:|https:"
+```
+
+The default inclusion pattern should be sufficient for most use cases, but you can customize it to be more specific about which files are included in your coverage report.
+
+Files that contain `test.js`, `test.ts`, `test.jsx`, or `test.tsx` in their name are excluded by default.
+
+This is equivalent to:
+
+```bash
+deno coverage --exclude="test\.(js|mjs|ts|jsx|tsx)$"
+```
+
+This default setting prevents your test code from contributing to your coverage report.
+For a URL to match it must match the include pattern and not match the exclude pattern.
+
+## Output Formats
+
+By default we support Deno's own coverage format - but you can also output coverage reports in the lcov format.
+
+```bash
+deno coverage --lcov --output=cov.lcov
+```
+
+This lcov file can be used with other tools that support the lcov format.
```

Comment:
What about `html` format?

Reply:
I've added in an example out outputting in html